terraform/paas/production.env.tfvars

```diff
@@ -4,4 +4,14 @@ paas_app_application_name  = "get-into-teaching-app-prod"
 paas_redis_1_name          = "get-into-teaching-prod-redis-svc"
 paas_additional_route_name = "beta-getintoteaching"
 instances                  = 2
-alerts                     = {}
+alerts = {	
+  GiT_App_Production_Healthcheck = {	
+    website_name  = "Get Into Teaching Website (Production)"	
+    website_url   = "https://beta-getintoteaching.education.gov.uk/healthcheck.json"	
```

Comment:
Is healthcheck the correct endpoint to hit?

Currently it
1. Tests for connectivity to Redis
2. Does a query against the API -> which will in turn than do a query against Gitis CRM
3. If either of the above fail, that is reflected in the returned JSON _but not the status code_ - it will still be 200

If the response from either Redis (unlikely) or the CRM (much more likely) is slow and times out - than the health check will still report 200 -> and StatusCake wont report the error

I think we should either
1) not be bothering to query the CRM every 60 seconds - ie use a different endpoint which just checks if the app is up
2) or we should be returning a different HTTP Status Code for `healthcheck.json` when its degraded

Reply:
I will raise a Trello ticket